src/lib/viewers/media/DashViewer.js

```diff
@@ -94,10 +96,16 @@ class DashViewer extends VideoBaseViewer {
         this.textTracks = []; // Must be sorted by representation id
         this.audioTracks = [];
 
+        this.videoAnnotationsEnabled = this.featureEnabled(VIDEO_ANNOTATIONS_ENABLED);
+
         // dash specific class
         this.wrapperEl.classList.add(CSS_CLASS_DASH);
     }
 
+    useReactControls() {
```

Comment:
Do we want to consider another name here? in React functions that start with `use` usually refer to hooks

Reply:
useReactControls is how that's referred to in the codebase and I just extracted that call into it's own function for consistency.